### PR TITLE
Add tabletools to recent R easyconfigs

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--skip"

File: easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb [hunk 1]
```diff
@@ -78,9 +78,6 @@ exts_default_options = {
 # !! order of packages is important !!
 # packages updated on 17th March 2023
 exts_list = [
-    # include packages that are part of the base installation of R,
-    # both to make sure they are available (via sanity check),
-    # and to be able to pass the check for required dependencies when installing extensions in parallel
```
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb]: Done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (5 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/85852c6209de061f3794528f86014e2d for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**